### PR TITLE
Bug 1915473: Annotate manifests for single-node-developer cluster profile

### PR DESCRIPTION
**Pull request by guillaumerose**
This implements phase 1 of openshift/enhancements#482

### CODE REVIEW

File: manifests.rhel/0000_90_cluster-update-keys_configmap.yaml [hunk 1]
```diff
@@ -66,3 +66,4 @@ metadata:
   creationTimestamp: null
   name: release-verification
   namespace: openshift-config-managed
+  namespace: openshift-config-managed
```
Comment (crawford) [inline manifests.rhel/0000_90_cluster-update-keys_configmap.yaml:69]: Duplicate

Comment (guillaumerose) [reply manifests.rhel/0000_90_cluster-update-keys_configmap.yaml]: sorry, fixed.

**Comment by guillaumerose** [comment]
/hold

CI is unhappy

**Comment by crawford** [comment]
CI is happy!

/hold cancel

**Pull request merged**